File: Java/Driver.java
class Driver extends Account{
    Integer id;
    String email;
    String password;

    public Driver(Integer id, String email, String password, String name, String document){
        super(name, document);
        this.id = id;
        this.email = email;
        this.password = password;
    }

    void printDataDriver(){
        System.out.println("ID: "+ id + " Email: " + email + " Password: " + password + " Name: " + name + " Document: " + document);
    }

}

File: JavaScript/Driver.js
class Driver extends Account{
    constructor(name, document, id, email, password){
        super(name, document);
        this.id = id;
        this.email = email;
        this.password = password;
    }

    printDataDriver(){
        console.log(this.name);
        console.log(this.document);
        console.log(this.id);
        console.log(this.email);
        console.log(this.password);
    }

}